pkg/bpftrace/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM linuxkit/alpine:ace75d0ec6978762d445083d6c8c6336bdb658ed
+RUN apk add --update \
+    bison \
+    build-base \
+    clang-dev \
+    clang-static \
+    cmake \
+    elfutils-dev \
+    flex-dev \
+    git \
+    linux-headers \
+    llvm5-dev \
+    llvm5-static \
+    zlib-dev
+
+# Put LLVM directories where CMake expects them to be
+RUN ln -s /usr/lib/cmake/llvm5 /usr/lib/cmake/llvm
+RUN ln -s /usr/include/llvm5/llvm /usr/include/llvm
+RUN ln -s /usr/include/llvm5/llvm-c /usr/include/llvm-c
+
+ENV GITREPO=https://github.com/iovisor/bpftrace.git
+ENV COMMIT=8f7f8214d7dd7bc25b7740a3c0e9a580a89e0244
+ENV STATIC_LINKING=on
+ENV RUN_TESTS=0
+
+RUN git clone ${GITREPO} /bpftrace \
+    && cd /bpftrace/ \ 
+    && mkdir -p /bpftrace/build \
+    && git checkout ${COMMIT}
+
+ENTRYPOINT ["/bin/sh", "/bpftrace/docker/build.sh", "/bpftrace/build", "Release" ]
```

Comment:
Should this really be an `ENTRYPOINT` or another command in the previous `RUN`? I think it's the latter.

And then if it is in fact a fully static binary, then the final stanzas can be

```
FROM scratch
WORKDIR
ENTRYPOINT[]
## Assuming the real value of the path to bpftrace
COPY --from=build /bpftrace/build/bpftrace /bpftrace
CMD ["/bpftrace"]
```

and the first can be `FROM linuxkit/alpine:ace75d0ec6978762d445083d6c8c6336bdb658ed AS build`

Reply:
Great suggestion. I'll try playing around with this.

Reply:
I've implemented this in the next commit.